src/ser.rs

```diff
@@ -341,15 +370,23 @@ impl<'s> ser::Serializer for &'s Serializer {
     }
 
     fn serialize_none(self) -> Result {
-        Ok(JsValue::UNDEFINED)
+        Ok(if self.serialize_none_as_null {
```

Comment:
Given that those grow up a bit, perhaps just delegate to `self.serialize_unit()` here so that the compiler could merge functions more easily.

Reply:
Done. And dropped `serialize_none_as_null` option. Should we change the name of `serialize_unit_as_null` to reflect that it changes none as well?

Reply:
Oh right, my bad, I forgot those are two different options. Maybe they should be separate after all... OTOH I think it should be fine to treat them in the same way. Let's rename to `serialize_missing_as_null`?

Reply:
Yes, I don't see a real need for them to be separated. I changed it to `serialize_missing_as_null`.